src/views/v2/tickets/footer/return-to-cart.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Block: Tickets
+ * Footer "Return to cart"
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/tickets/footer/return-to-cart.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ * @version TBD
+ *
+ * @var string                  $cart_url     The provider cart URL.
+ * @var string                  $checkout_url The provider checkout URL.
+ * @var bool                    $is_mini      True if in "mini cart" context.
+ */
+
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 102dece38962f9466f36a97cd4368b12</sub>